src/AppInstallerCLITests/InstallerMetadataCollectionContext.cpp

```diff
@@ -0,0 +1,517 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "TestCommon.h"
+#include "TestSource.h"
+
+#include <winget/InstallerMetadataCollectionContext.h>
+
+using namespace AppInstaller;
+using namespace AppInstaller::Repository;
+using namespace AppInstaller::Repository::Correlation;
+using namespace AppInstaller::Repository::Metadata;
+using namespace TestCommon;
+
+namespace
+{
+    // Indicates to set minimal defaults on TestInput
+    struct MinimalDefaults_t {} MinimalDefaults;
+
+    struct TestInput
+    {
+        TestInput() = default;
+
+        TestInput(MinimalDefaults_t)

```

Comment:
>TestInput [](http://example.com/codeflow?start=8&length=9)

Should we have an input and output test to showcase what the input json and output json looks like?

Reply:
Do you mean a test case with a literal string input and verification as a string output looking for literal values?